src/DatabaseManager.php

```diff
@@ -215,12 +152,7 @@ public function rollBack()
         return true;
     }
 
-    /**
-     * @param string|null $name
-     *
-     * @return mixed
-     */
-    public function getLastInsertId($name = null)
+    public function getLastInsertId(string $name = null)
```

Comment:
Are we 100% sure this works? Null is definitely different than string. I haven't tested something like this yet in PHP 7.